### PR TITLE
With Hector's changes

### PR DESCRIPTION
**Pull request by eklein-okta**
@cwu-okta @danielzeller-okta @vladdha-okta @mystiberry-okta Please check this PR.  As soon as it's okay, we can rebuild the site and send it out for a more general review.

### CODE REVIEW

File: _source/_docs/how-to/updating_saml_cert.md [hunk 1]
```diff
@@ -307,9 +307,34 @@ If you have OpenSSL installed, from the command line run:
 
 Where:
 
-`<your certificate>` is the certificate filename relative to the current directory.
-
-Use a free CSR and Certificate Decoder service and enter the contents of your certificate. These tools are readily found through a web search.
+`<your certificate>` is the certificate filename relative to the current directory. The certificate must be in PEM format. If you saved the
```
Comment (danielzeller-okta) [inline _source/_docs/how-to/updating_saml_cert.md]: At this point we have not called out how to save the certificate. It must be saved in plaintext with the Begin Certificate and End Certificate lines added. We should call out that they should use a plaintext editor like notepad/textedit. If they were to use something like Word/Openoffice openssl will not accept it.

File: _source/_docs/how-to/updating_saml_cert.md [hunk 2]
```diff
@@ -307,9 +307,34 @@ If you have OpenSSL installed, from the command line run:
 
 Where:
 
-`<your certificate>` is the certificate filename relative to the current directory.
-
-Use a free CSR and Certificate Decoder service and enter the contents of your certificate. These tools are readily found through a web search.
+`<your certificate>` is the certificate filename relative to the current directory. The certificate must be in PEM format. If you saved the 
+certificate text from the `x5c` element returned from an API call, you must add the **Begin Certificate** and **End Certificate** lines with the 
+hyphens to the top and bottom of the file, as shown below. (The certificate shown below has been altered and is not valid.)
```
Comment (danielzeller-okta) [inline _source/_docs/how-to/updating_saml_cert.md]: Is there a specific reason that the cert's that we're showing in the document are invalid? To me this detracts from the authenticity of the document.

Comment (eklein-okta) [inline _source/_docs/how-to/updating_saml_cert.md]: As I understand, it's a temporary measure. But I'll leave that call to Cindy.

File: _source/_docs/how-to/updating_saml_cert.md (hunk 1 above)
Comment (eklein-okta) [inline _source/_docs/how-to/updating_saml_cert.md]: I just added the plaintext editor part...

**Comment by danielzeller-okta** [comment]
A couple of comments:

- In step one the admin is going to see "Use the _List Apps_ to return a list of all apps." where _List Apps_ is a link to the apps api. I'd suggest changing the link text to "List Apps Api" or just "Apps Api"
- There are no instructions on how to make the api calls. We need to either link to existing postman instructions or provide it inline.
- The request on step 3 where we are updating the app needs to be `PUT` instead of `POST`.
- The link in the last section, "Obtaining the Certificate for an App from a URL", is incorrect. The subdomin should be "\<theirOrgSubdomin\>-admin"

**Comment by eklein-okta** [comment]
@danielzeller-okta Done!

**Comment by cwu-okta** [comment]
When describing how to create a certificate to use with openssl we indicate to use a plain text editor and also add the begin/end cert tags. We should also let users know you cannot add trailing white spaces (space, carriage return, etc) at the end of the file. Adding a trailing carriage return (for example) will result in an invalid certificate which openssl will not be able to use.

**Comment by eklein-okta** [comment]
@cwu-okta Done!

**Comment by cwu-okta** [comment]
🚢

**Comment by eklein-okta** [comment]
Yay!

**Pull request merged**